lib/fastlane/setup/setup_ios.rb

```diff
@@ -72,6 +79,29 @@ def generate_appfile
       Helper.log.info "Created new file '#{path}'. Edit it to manage your preferred app metadata information.".green
     end
 
+    def default_run_produce
+      Helper.log.info "Running produce..."
```

Comment:
yeah, i agree this is pretty weird. i think that i thought we had talked about this in one of our initial meetings about this and thought it should happen, i am happy to get rid of this part of the feature.